packages/node_modules/@ciscospark/media-engine-webrtc/browsers.js

```diff
@@ -0,0 +1,53 @@
+// eslint-disable-next-line strict
+'use strict';
+
+/* eslint camelcase: [0] */
+// const path = require(`path`);
+module.exports = function createBrowsers() {
+  if (process.env.SC_TUNNEL_IDENTIFIER) {
+    return {
+      sl_chrome_latest_osx12: {
+        base: `SauceLabs`,
+        platform: `OS X 10.12`,
+        browserName: `chrome`,
+        version: `latest`,
+        chromeOptions: {
+          args: [
+            `--use-fake-device-for-media-stream`,
+            `--use-fake-ui-for-media-stream`
+          ]
+        }
+      },
```

Comment:
we haven't setup the windows openh264 stuff yet; once we have that, we'll add it here.